Gems/LmbrCentral/Code/Source/Shape/EditorCapsuleShapeComponent.cpp

```diff
@@ -84,10 +84,18 @@ namespace LmbrCentral
         AzFramework::DebugDisplayRequests& debugDisplay)
     {
         DisplayShape(
-            debugDisplay, [this]() { return CanDraw(); },
+            debugDisplay,
+            [this]()
+            {
+                return CanDraw();
+            },
             [this](AzFramework::DebugDisplayRequests& debugDisplay)
             {
-                DrawShape(debugDisplay, { m_shapeColor, m_shapeWireColor, m_displayFilled }, m_capsuleShapeMesh);
+                DrawShape(
+                    debugDisplay,
+                    { m_shapeColor, m_shapeWireColor, m_displayFilled },
```

Comment:
nit: Could we add the type before the `{...}` object created here as I've found searching for instances of it is made much easier when the type is included (this again is a fault of mine from the past and I've since changed my ways 😅)